packages/playwright/src/runner/runner.ts

```diff
@@ -83,14 +82,13 @@ export class Runner {
       createLoadTask('in-process', { failOnLoadErrors: true, filterOnly: false }),
       createReportBeginTask(),
     ] : [
+      createApplyRebaselinesTask(),
```

Comment:
it looks odd that it goes first in the list, maybe add a comment about teardown ?